Chapter5/stats_handler.erl

```diff
@@ -0,0 +1,13 @@
+-module(stats_handler).
+-export([init/1, terminate/1, handle_event/2]).
+
+init(Db) -> Db.
+terminate(Db) -> {db, Db}.
+
+handle_event({Type, _Id, Description}, Db) ->
+	case lists:keysearch( {Type, Description}, 1, Db ) of
+		false ->
+			Db ++ [ {{Type, Description}, 1}] ;
```

Comment:
Try to avoid adding element to the head of the list, see [comment](https://github.com/danieljharris/learning-erlang/pull/1#discussion_r132897832) for more info.